common/flatpak-run.c

```diff
@@ -752,6 +752,35 @@ flatpak_run_apply_env_vars (FlatpakBwrap *bwrap, FlatpakContext *context)
     }
 }
 
+gboolean
+flatpak_run_apply_command (FlatpakBwrap *bwrap, FlatpakContext *context)
+{
+  GHashTableIter iter;
+  gpointer key, value;
+
+  g_hash_table_iter_init (&iter, context->command);
+  if (!g_hash_table_iter_next (&iter, &key, &value))
+    return FALSE;
+
+  const char *val = value;
+  gint argc;
+  g_auto(GStrv) argv = NULL;
+
+  if (!g_shell_parse_argv (val, &argc, &argv, NULL))
+    return FALSE;
+
+  flatpak_bwrap_add_args (bwrap, "--", argv[0], NULL);
+
+  int i;
+
+  for (i = 1; i < argc; i++)
+    {
+      flatpak_bwrap_add_arg (bwrap, argv[i]);
+    }
```

Comment:
This could be simplified into a loop starting at 0, without the special case for `[0]`.